guide/inheritance-wallet/onboarding-cosigners.md

```diff
@@ -0,0 +1,156 @@
+    - file: alice-import-config/alice-wallet-success
+      alt: Screen showing a success message and the option to download a wallet backup kit. 
+      caption: On the success screen the app prompts Alice to download the wallet backup kit. She will do that later.
+    - file: alice-import-config/app-home-done
+      alt: App home screen showing the newly created wallet.
+      caption: Back on the home screen, Alice sees that she should download the backup kit and enable her signing device.
+images_alice-enable-device:
+    - file: alice-enable-device/enable-alice-home
+      alt: Home screen showing the wallet with two tasks associated.
+      caption: Alice is ready to enable her BitBox to sign transactions for the wallet. 
+    - file: alice-enable-device/enable-alice-dialog
+      alt: Scren showing an informational dialog about enabling signing devices.
+      caption: The app tells her what this step is about and how it works.
+    - file: alice-enable-device/enable-alice-wallet-export
+      alt: Screen showing different options to export the wallet configuration.
+      caption: Alice connects her BitBox via USB-C to export the wallet configuration.
+    - file: alice-enable-device/enable-alice-device-flow
+      alt: Placholder screen representing the device-specific wallet activation flow.
+      caption: The app guides her throught the import flow on her BitBox.
+    - file: alice-enable-device/enable-alice-success
+      alt: Success screen showing the option activate another device or go to the home screen.
+      caption: On the success screen, the app shows the option to enable another signing device.
+    - file: alice-enable-device/enable-alice-completion
+      alt: Home screen showing the wallet with two tasks associated.
+      caption: On the home screen, Alice checks off the task. 
+    - file: alice-enable-device/enable-alice-completion-dialog
+      alt: Screen showing a confirmation dialog.
+      caption: The app asks whether she has enabled all signing devices.
+    - file: alice-enable-device/enable-alice-done
+      alt: Home screen showing the wallet with only one task associated.
+      caption: The last thing to do is to download the wallet backup kit.
+---
+
+<!--
+
+Editor's notes
+
+This page covers how Alice imports the savings wallet to her own app and enables her signing device to create and sign transactions.  
+
+Illustration sources
+
+https://www.figma.com/file/h5GP5v5dYfpXXfEUXf6nvC/Family-inheritance-wallet?type=design&node-id=5542%3A2119&mode=design&t=sBtcvrDzb8MPtWaK-1
+
+-->
+
+# Cosigner onboarding
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+[So far]({{ '/guide/inheritance-wallet/wallet-creation/' | relative_url }}), the wallet has been created on Bob's app. The next step is to get Alice set up as a cosigner and to enable her to sign transactions. To do this Alice needs to:
```

Comment:
```suggestion
In the previous step, Bob [created the wallet]({{ '/guide/inheritance-wallet/wallet-creation/' | relative_url }}) on his device. The next step is to get Alice set up as a cosigner and to enable her to sign transactions. To do this Alice needs to:
```
```suggestion
[So far]({{ '/guide/inheritance-wallet/wallet-creation/' | relative_url }}), the wallet has been created on Bob's app. The next step is to get Alice set up as a cosigner and to enable her to sign transactions. To do this Alice needs to:
```

Reply:
For readability, feels a little strange to start the page with a link to 'So far', as it's unclear what will be behind the link.